serve/mlc_serve/model/paged_cache_model.py

```diff
@@ -1,602 +1,26 @@
-import math
-import os
-from typing import List, Union, Optional
+from typing import Union
 from pathlib import Path
-
 import structlog
-import numpy as np
-import torch
-import tvm
-from tvm import relax
-from tvm.runtime import disco as di
 
 from .base import get_model_artifact_config
-from .paged_cache_manager import KVCache, CacheManager
+from .paged_cache_manager import CacheManager
 from .tokenizer import HfTokenizerModule, ConversationTemplate, Tokenizer
-from ..engine import (
-    SamplingType,
-    MLCServeEngineConfig,
-    SamplingParams,
-    SequenceId,
-    PROMPT_SEQEUNCE_INDEX,
-    get_prompt_sequence_id,
-)
+from .tvm_model import init_tvm_model
```

Comment:
If we assume serving layer can/will execute other frameworks/inference engines models we cannot  require tvm to be installed everywhere and in such environments, importing of tvm will cause crash. We need to put importing of tvm_model in try/catch block.

Reply:
TVM is still our primary model, and other implementations can be used as a fallback when TVM model is not ready or performant. We assume that TVM is always available.

Reply:
Not sure I catch the idea. If we decide to use trt-llm for certain model, it will be run in the environment having libraries for dependent framework. No TVM will be required. If tvm model can be loaded, it will be explicitly pointed in the config

Reply:
Hi, @elvin-n. The main idea of having a torch based model is to have a fast deployment path with the low squeeze. We will follow-up with TVM model for the high squeeze. TRT-LLM is usually slow in expanding official model support so we would need to rewrite the model with TRT-LLM constructs, which will also require the non-trivial efforts in the model integration. So this will be a good experiment to see if we can land such workflow: low squeeze with torch model for the fast deployment -> high squeeze with TVM to catch up.